src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
The import statement for Random is not necessary if you use a single instance of Random as a class-level variable.